### PR TITLE
Add read-more to discussion

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/1157

### CODE REVIEW

File: udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.vue [hunk 1]
```diff
@@ -67,36 +61,37 @@ export default defineComponent({
 
     const containerHeight = ref(DEFAULT_HEIGHT);
 
-    const updateReadMoreHeight = () => {
-      let contentHeight = Array.from(container.value?.children || [])
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.vue:71]: Some changes here in the logic.
I think I had some issues expanding: it did not expand to the entire description or message.
May be related, but I'm not sure that toggles introduced in markdown are being treated correctly.

File: udata_front/theme/gouvfr/assets/js/components/discussions/threads.vue [hunk 2]
```diff
@@ -1,15 +1,3 @@
-<!--
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/discussions/threads.vue:1]: You should probably rename this file. The storybook shows:
```
Failed to resolve import "./Threads.vue" from "udata_front/theme/gouvfr/assets/js/components/discussions/Threads.stories.js". Does the file exist?
```

File: udata_front/theme/gouvfr/assets/js/components/discussions/Thread/Thread.vue [hunk 3]
```diff
@@ -72,72 +74,89 @@
...
+      return window.location.origin + window.location.pathname + hash;
+    }
+
+    const displayForm = () => {
+      auth();
+      showForm.value = true;
+    }
+
+    const replyToThread = (comment) => {
+      return api
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/discussions/Thread/Thread.vue:140]: I think you need to import it

**Review by maudetes — COMMENTED** [review]
Thank you! It's quite nice to test out the storybook :)

File: udata_front/theme/gouvfr/assets/js/components/discussions/threads.vue (hunk 2 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/discussions/threads.vue:1]: Case insensitive file system are the worst. I'm renaming it on Github.

File: udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.stories.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  component: ReadMore,
+};
+
+export const DefaultReadMore = {
+  render: (args) => ({
+    components: { ReadMore },
+    setup() {
+      return { args };
+    },
+    template: ` <ReadMore v-bind="args" max-height="">
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.stories.js:14]: Xnapper-2023-11-10-10 13 42

With a large side bar (or web inspector), it should be activated. Also it is reactive now so a page size change should be detected.

File: udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.vue (hunk 1 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.vue:71]: Yep, I forgot to change the ref on some part of the logic

**Review by maudetes — APPROVED** [review]
:ok_hand:

**Pull request merged**